packages/react/src/icons/IconAlertCircle.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import styles from './Icon.module.css';
+
+interface IconProps {
+  ariaLabel?: string;
+  ariaLabelledby?: string;
+  ariaHidden?: boolean;
+  className?: string;
+  color?: string;
+  size?: 'xs' | 's' | 'm' | 'l' | 'xl';
+  style?: React.CSSProperties;
+}
+
+export const IconAlertCircle = ({
+  ariaLabel = 'alert-circle',
+  ariaLabelledby,
+  ariaHidden = true,
+  className = '',
+  color,
+  size = 's',
+  style = {},
+}: React.SVGProps<SVGSVGElement> & IconProps) => (
+  <svg
+    className={[styles.icon, styles[size], className].filter((e) => e).join(' ')}
+    role="img"
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 24 24"
+    aria-label={ariaLabel}
+    aria-labelledby={ariaLabelledby}
+    aria-hidden={ariaHidden}
+    color={color}
+    style={style}
+  >
+    <path
+      fillRule="evenodd"
+      clipRule="evenodd"
+      d="M12 2C17.5228 2 22 6.47715 22 12C22 17.5228 17.5228 22 12 22C6.47715 22 2 17.5228 2 12C2 6.47715 6.47715 2 12 2ZM12 4C7.58172 4 4 7.58172 4 12C4 16.4183 7.58172 20 12 20C16.4183 20 20 16.4183 20 12C20 7.58172 16.4183 4 12 4ZM13 16V18H11V16H13ZM13 6V14H11V6H13Z"
+      fill="currentColor"
+    ></path>
```

Comment:
Building hds-react gives error on every icon component:

`35:5  error  Empty components are self-closing                                  react/self-closing-comp
  40:6  error  Closing tag of a multiline JSX expression must be on its own line  react/jsx-closing-tag-location`
  
I think this is simply fixed by having the path be self-closing on a new line.